src/runtime/config/configuration-qemu-nvidia-gpu-snp.toml.in

```diff
@@ -698,3 +698,7 @@ create_container_timeout = @DEFAULTTIMEOUT_NV@
 # to the hypervisor.
 # (default: /run/kata-containers/dans)
 dan_conf = "@DEFDANCONF@"
+
+# Enforce guest pull. This instructs the runtime to communicate to the agent via annotations that
+# the container image should be pulled in the guest, without using an external snapshotter.
+force_guest_pull = @DEFFORCEGUESTPULL@
```

Comment:
I'd really like to have this as `experimental_force_guest_pull`, which would allows us to easily drop it whenever we can use containerd's image transfer API (which would be the "ideal" way to handle this).

If you can add the experimental prefix, and also some notes about this being a stopgapper till we have an ideal solution in place, that would be awesome.

Reply:
done